sklearn/datasets/base.py

```diff
@@ -255,6 +255,12 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+    ----------
+    return_X_y : boolean, default=False.
+        If true, you will get (X, y), else you will get Bunch.
+
+
     Returns
     -------
     data : Bunch
```

Comment:
Documentation for python is generally not in first and second person. You probably want to say

```
return_X_y : bool, default=False
    If True, returns (X, y) instead of a Bunch object.